README.md

```diff
@@ -117,7 +130,7 @@ This endpoint allows you to create a new order.
 ##### Data Params
 ```json
         {
-        "name": "Ashley Coles",
+                "name": "Ashley Coles",
```

Comment:
please remove some of the indentation so that the curly braces are against the left side of the page and so that the object properties are in line

Reply:
it shows up weird in the code but in the actual readme it works, not srue why